checklist_plus/text_generation/masked_lm.py

```
@@ -1,335 +1,287 @@
+            self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+            # self.tokenizer = BertTokenizer.from_pretrained('bert-base-cased')
+            # self.model = BertForMaskedLM.from_pretrained('bert-base-cased')
+            self.tokenizer = AutoTokenizer.from_pretrained(model_name)
+            self.model = AutoModelForMaskedLM.from_pretrained(model_name)
+            self.model.to(self.device)
+            self.model.eval()
+            self.prefix_sentence = prefix_sentence
+            self.prefix_len = len(self.tokenizer.encode(prefix_sentence, add_special_tokens=False))
+            self.allow_word_pieces = allow_word_pieces
+            tmp = self.tokenizer.tokenize(' a')[0].split('a')
+            assert len(tmp) == 2
```

Ruff (0.12.2)

Use of `assert` detected

(S101)